src/org/project/ShoppingCLI.java

```diff
@@ -81,8 +82,8 @@ public void removeProductFromCart() {
             try {
                 cart.removeProductFromCart(availableProducts.get(productNumberToRemove - 1).getId(), quantityToRemove);
                 System.out.println("Product removed from cart.");
-            } catch (IllegalArgumentException e) {
-                System.out.println(e.getMessage());
+            } catch (ProductNotAvailableException e) {
```

Comment:
no ale to one sa unchecked po to zeby ich nie lapac, nie chcemy miec intearkcji na nie tylko chcemy rzucac do uzytkownika po prostu -> ten try catch niepotrzebny imo, rzucenie runtime po naszym customowym tez nie jest poprawnym podejsciem, po to mamy nasz customowy zeby bardziej konkretnie rzucac a nie runtimeExc ogolny